metal3-dev/local-cbo.sh

```diff
@@ -8,6 +8,21 @@ source $SCRIPTDIR/logging.sh
 source $SCRIPTDIR/common.sh
 source $SCRIPTDIR/utils.sh
 
+# Stop already running CBO.
+pushd $SCRIPTDIR
+$SCRIPTDIR/stop-cbo.sh
```

Comment:
nit: I would keep in line with https://github.com/openshift-metal3/dev-scripts/blob/21ec5582c73c4887f7bc85f5de079a2620a24b3b/metal3-dev/local-bmo.sh#L24-L27